webpack/app/js/tile.js

```diff
@@ -69,4 +65,47 @@ export default class AgendaTile {
       },
     });
   }
+  updateDayPicker(data) {
```

Comment:
please document this.

Reply:
the method name is clear enough

Reply:
is not clear enough; the method name don't say anything on what are you doing here.